io.openems.edge.ess.cluster/src/io/openems/edge/ess/cluster/EssCluster.java

```diff
@@ -156,6 +161,20 @@ private void calculateChannelValues() {
 			}
 		}
 
+		// Setting the allowed charge and discharge power
+		for (String essId : this.config.ess_ids()) {
+			ManagedSymmetricEss ess;
+			try {
+				ess = this.componentManager.getComponent(essId);
```

Comment:
Careful: If ess is not a `ManagedSymmetricEss` this will give a ClassCastException. I'd rather add it to the above for-loop.